### PR TITLE
closes #98 reference page@devel

### LINKED ISSUE #98: Documentation:  Bring our reference page in line with admiral new approaches
### Please select a category the issue is focused on?

Other

### Let us know where something needs a refresh or put your idea here!

See https://github.com/pharmaverse/admiral/issues/1105

### PR DESCRIPTION
**Pull request by rossfarrugia**
Closes #98.

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@sgorm123 probably best for you to review this one as you need to be aware of the updates for your ongoing issue anyway and probably best to merge this with your branch once its in `devel` - the background is explained here https://github.com/pharmaverse/admiral/issues/1105

File: R/tte_source.R [hunk 1]
```diff
@@ -41,7 +42,7 @@ death_event <- event_source(
   )
 )
 
-#' @keywords tte_source
+#' @keywords source_specifications
 #' @rdname tte_source_objects
```
[the remark this answers is not in the document]
Comment (rossfarrugia) [reply R/tte_source.R:46]: think we only need it once here for these objects - i used same convention admiral core used at https://github.com/pharmaverse/admiral/blob/devel/R/tte_sources.R

File: R/derive_param_confirmed_bor.R [hunk 2]
```diff
@@ -501,7 +503,8 @@ derive_param_confirmed_bor <- function(dataset,
 #'
 #' @author Stefan Bundfuss
 #'
-#' @keywords user_utility
+#' @family move_adm_dev
+#' @keywords move_adm_dev
```
[the remark this answers is not in the document]
Comment (rossfarrugia) [reply R/derive_param_confirmed_bor.R:507]: yeh good spot, should be `utils_fmt = "Utilities for Formatting Observations: "` as we're keeping this function in admiralonco

File: man/roxygen/meta.R [hunk 3]
```diff
@@ -0,0 +1,8 @@
+list(
+  rd_family_title = list(
+    der_prm_adrs = "ADRS Functions for adding Parameters: ",
+    utils_fil = "Utilities for Filtering Observations: ",
+    source_specifications = "Source Specifications: ",
```
[the remark this answers is not in the document]
Comment (rossfarrugia) [reply man/roxygen/meta.R:5]: I'd rather keep consistent with admiral core choices here, for user familiarity across the packages, and for me the place people would find most on this topic would be the ADTTE vignette that covers in detail

File: R/tte_source.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (sgorm123) [reply R/tte_source.R:46]: Then do we not get the SeeAlso functionality with this approach?

[the remark this answers is not in the document]
Comment (rossfarrugia) [reply R/tte_source.R:46]: they all appear on the same user guide page _(and in a table in the ADTTE vignette too)_, e.g. see for example: https://pharmaverse.github.io/admiral/reference/tte_source_objects.html

**Review by sgorm123 — APPROVED** [review]

**Pull request merged**